### ISSUE TITLE
Option for original languages

### ISSUE THREAD

**Issue by absolutsoul** [aschmelyun/subvert#27]
Add an option to select not to translate the original language.

**Comment by aleclerc7** [comment]
Indeed, when selecting another language, the results are in English. It would be better if the result was in the intended language.  Thank you very much! 😊

**Comment by aleclerc7** [comment]
I just saw in the code that if in fact you **do not select a language**, it will not be translated, but stay in the original language.

So the real change that should be made is to **change the prompt from “Video Language” to “Translate to”** which is exactly what is being asked to ChatGPT. (See https://github.com/aschmelyun/subvert/blob/main/src/app/Actions/TranslateSubtitles.php)

Kind regards.
